<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<!-- <node pkg="rosbag" type="play" name="player" args="&#45;&#45;clock /home/amsl/ozaki/bagfiles/infant/mocap/0127_infant/2019&#45;01&#45;27&#45;18&#45;38&#45;20.bag &#45;s 30"/> -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/sq2/outdoor_dkan/2018-12-11-13-37-56.bag -r 1"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find gyrodometry)/rviz_config/gyrodometry3d.rviz"/>

	<!-- semi sensor driver -->
	<node pkg="msg_conversion" name="infant_odometry_republish" type="infant_odometry_republish">
		<!-- <param name="velocity_scalar_ratio" type="double" value="0.92"/> -->
	</node>

	<!-- main -->
	<node pkg="gyrodometry" name="initial_alignment" type="initial_alignment" output="screen">
		<!-- <param name="ini_ori_is_0001" type="bool" value="false"/> -->
		<!-- <param name="frame_id" type="string" value="/odom"/> -->
		<!-- <param name="child_frame_id" type="string" value="/initial_orientation"/> -->
		<!-- <param name="min_record_size" type="int" value="100"/> -->
		<!-- <param name="max_duration" type="double" value="60.0"/> -->
		<!-- <param name="th_linear_deflection" type="double" value="0.03"/> -->
		<!-- <param name="th_angle_deflection" type="double" value="0.2"/> -->
	</node>
	<node pkg="gyrodometry" name="gyrodometry3d" type="gyrodometry3d" output="screen">
		<!-- <remap from="/odom" to="/tinypower/odom/republished"/> -->
		<!-- <param name="wait_inipose" type="bool" value="true"/> -->
		<!-- <param name="linear_vel_is_available" type="bool" value="true"/> -->
		<!-- <param name="frame_id" type="string" value="/odom"/> -->
		<!-- <param name="child_frame_id" type="string" value="/gyrodometry"/> -->
	</node>
</launch>
